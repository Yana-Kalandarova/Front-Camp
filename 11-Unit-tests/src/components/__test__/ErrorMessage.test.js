import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { mount } from 'enzyme';
import ErrorMessage from '../ErrorMessage';

describe('ErrorMessage', () => {
  it('renders correctly', () => {
    expect(
      mount(
        <BrowserRouter>
          <ErrorMessage message={'mock message'} />
        </BrowserRouter>
      )
    ).toMatchSnapshot()
  })
})
