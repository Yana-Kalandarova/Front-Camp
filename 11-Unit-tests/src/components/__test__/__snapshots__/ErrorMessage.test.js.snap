// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correctly 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <ErrorMessage
      message="mock message"
    >
      <p
        className="contact_form-error"
      >
        mock message
      </p>
    </ErrorMessage>
  </Router>
</BrowserRouter>
`;
